graph.go

```diff
@@ -306,9 +311,13 @@ func doGraph(buildPkg *build.Package) (*gog.Output, error) {
 	allGoFiles = append(allGoFiles, buildPkg.GoFiles...)
 	allGoFiles = append(allGoFiles, buildPkg.CgoFiles...)
 	allGoFiles = append(allGoFiles, buildPkg.TestGoFiles...)
-	if len(allGoFiles) == 0 {
-		return &gog.Output{}, nil
+
+	if !testPkg {
+		// graph non-test package
+		return doGraphFiles(fset, buildPkg.ImportPath, buildPkg.Dir, allGoFiles, dependencies)
 	}
+
+	// prepare type info for non-test package
```

Comment:
this comment doesn't really make sense, since if `!testPkg`, we return. Surely we are preparing for a test package only here?